README.md

```diff
@@ -8,41 +8,54 @@
 
 <img src="https://www.haskell.org/cabal/images/Cabal-light.png" align="right">
 
-This Cabal Git repository contains the following packages:
+This Cabal Git repository contains the following main packages:
 
  * [Cabal](Cabal/README.md): the Cabal library package ([license](Cabal/LICENSE))
  * [Cabal-syntax](Cabal-syntax/README.md): the `.cabal` file format library ([license](Cabal-syntax/LICENSE))
  * [cabal-install](cabal-install/README.md): the package containing the `cabal` tool ([license](cabal-install/LICENSE))
+ * [cabal-install-solver](cabal-install-solve): the package containing the solver component of the `cabal` tool ([license](cabal-install-solver/LICENSE))
 
 The canonical upstream repository is located at
 https://github.com/haskell/cabal.
 
-Installing cabal-install (by downloading the binary)
-----------------------------------------------------
+Ways To Get cabal-install Binary
+--------------------------------
 
-Prebuilt binary releases can be obtained from https://www.haskell.org/cabal/download.html.
-The `cabal-install` binary download for your platform should contain the `cabal` executable.
+1. _GHCup (**preferred**)_ Get GHCup using [the directions on its website](https://www.haskell.org/ghcup/) and run:
```

Comment:
Add colon `:` after item heading.
Applies to 5 items.

Reply:
Fair, and I added it. There's a (reasonably widely accepted) style for paragraph headings when you use different font face and, typically, extra horizontal space, and no colon. I am guilty in omitting the extra space part of this recipe more often than not. So, I stand corrected.